test/utils/Strings.test.js

```diff
@@ -29,12 +29,24 @@ contract('Strings', function (accounts) {
       '12345678901234567890123456789012345678901234567890',
       '123456789012345678901234567890123456789012345678901234567890',
       '1234567890123456789012345678901234567890123456789012345678901234567890',
-    ].reduce((acc, value) => Object.assign(acc, { [value]: new BN(value) }), {
-      MAX_UINT256: constants.MAX_UINT256.toString(),
+    ].reduce((acc, value) => Object.assign(acc, { [value]: new BN(value), [-value]: new BN(-value) }), {
```

Comment:
This is unwieldy but I want to tackle it separately by adding a linter rule to disallow `reduce`.